rstudio/rstudio
# Add i18n for all options menus

### Intent

See #9083 for more context on i18n efforts.
This PR enables i18n for all menus under Options->* (or Tools->Global Options on desktop). This excludes most of the values produced by the settings (preferences) json files, though there is some preliminary work there. The rest will be addressed in a separate PR.

### Approach

This is a larger PR as there were many UI components involved in these menus and widgets, most of which required updating the java code and creation of the corresponding constants interface and _en.properties file.
Implementation follows the same pattern from [#9485](https://github.com/rstudio/rstudio/pull/9485) as described in gwt/tools/i18n-helpers/README.md. Running the create_dev_locale.sh will create all the new "dev" locale from any new "en" properties files for easier visual confirmation. Refer to [README](https://github.com/rohank07/rstudio/blob/rohan/i18n-all-options-menu/src/gwt/tools/i18n-helpers/README.md) for more information.

### Automated Tests

No additional testing is added. We are interested in feedback on what testing would be appropriate.

### QA Notes

This change is believed to be fully working without changing the user experience, but feedback is welcome. To confirm i18n is working for the the Options->* menus, while running RStudio in devmode you can do one of:
Modify an entry in one of the new _en.properties files, and refresh RStudio with ant in devmode.
Run create_dev_locale.sh to clone all _en properties files, which will add a prefix of "@" to each constant and message. Then open RStudio using http://localhost:8787/?locale=dev and browse to the Options->* menus. You can also view in desktop mode by swapping the locale element value attribute from value="en" to value="dev" in RStudio.gwt.xml.

### Checklist




cc @gtritchie
